mirrord/intproxy/protocol/src/lib.rs

```diff
@@ -372,6 +365,13 @@ impl_request!(
     res_path = ProxyToLayerMessage::File => FileResponse::ReadDir,
 );
 
+impl_request!(
+    req = ReadDirBatchRequest,
+    res = RemoteResult<ReadDirBatchResponse>,
+    req_path = LayerToProxyMessage::File => FileRequest::ReadDirBatch,
+    res_path = ProxyToLayerMessage::File => FileResponse::ReadDirBatch,
+);
```

Comment:
This is required only for the code to compile, right? Layer should not send batch requests

Reply:
Should I break the pattern? Kept it like this to have it looking like all the other requests.